vita3k/gui/src/license_install_dialog.cpp

```diff
@@ -97,8 +98,14 @@ void draw_license_install_dialog(GuiState &gui, HostState &host) {
         title = !indicator["install_complete"].empty() ? indicator["install_complete"] : "Installation complete.";
         ImGui::Spacing();
         ImGui::TextColored(GUI_COLOR_TEXT, "Successfully installed license.\nContent ID: %s\nTitle ID: %s", host.license_content_id.c_str(), host.license_title_id.c_str());
+        if (work_path)
+            ImGui::Checkbox("Delete the work.bin/rif file?", &delete_work_file);
```

Comment:
Wouldn't it be more user friendly to ask if delete the license rather than work./rif file?

Reply:
ah didn't check the context, its good then :+1: